// Program statements that you want to moniter for excepption are contained in
// a try block

// If any exception occurs within the try block, it is thrown(usig throw)

// The exception is caught, using catch, and processed












